hermes-consumers/src/test/groovy/pl/allegro/tech/hermes/consumers/consumer/offset/OffsetCommitterTest.groovy

```diff
@@ -353,11 +354,11 @@ class OffsetCommitterTest extends Specification {
         return new SubscriptionPartitionOffset(partitionOffset, offset)
     }
 
-    private assignPartitions(int... partitions) {
+    private assignPartitions(int ... partitions) {
```

Comment:
Looks strange, 'cause `...` are related to the type. Space looks suspicious 🤔

Reply:
agree, intellij default formatter, I missed it